Source/Core/DolphinQt/Main.cpp

```diff
@@ -221,8 +230,45 @@ int main(int argc, char* argv[])
   }
```

Comment:
Presumably you want the `!args.empty()` case to work the same way as the `exec` case.